qubesmanager/create_new_vm.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, app, qvm_collection, trayIcon, parent = None):
         self.fill_template_list()
         self.fill_netvm_list()
 
-        self.vmname.setValidator(QRegExpValidator(QRegExp("[a-zA-Z0-9-]*", Qt.CaseInsensitive), None))
+        self.vmname.setValidator(QRegExpValidator(QRegExp("[a-zA-Z0-9_-.]*", Qt.CaseInsensitive), None))
```

Comment:
`-` needs to be the last character in set (`[]`), otherwise it means a characters range.